WalletWasabi.Tests/UnitTests/GUI/CommandInterpreterTest.cs

```diff
@@ -0,0 +1,42 @@
+using Mono.Options;
+using System.IO;
+using WalletWasabi.Gui.CommandLine;
+using WalletWasabi.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.GUI
+{
+	public class CommandInterpreterTest
+	{
+		[Fact]
+		public async void CommandInterpreterShowsHelpAsync()
+		{
+			var textWriter = new StringWriter();
+			var c = new CommandInterpreter(textWriter);
+			await c.ExecuteCommandsAsync(new string[] { "wassabee", "--help" }, new Command("mixer"), new Command("findpassword"));
+
+			Assert.Equal(
+				$@"Wasabi Client Version: {Constants.ClientVersion}
+Compatible Coordinator Version: {Constants.ClientSupportBackendVersionText}
+Compatible Bitcoin Core and Bitcoin Knots Versions: {Constants.BitcoinCoreVersion}
+Compatible Hardware Wallet Interface Version: {Constants.HwiVersion}
+
+Usage: wassabee [OPTIONS]+
```

Comment:
This is weird.

Reply:
What is weird the name wassabee?

Reply:
No. `Usage: wassabee [OPTIONS]+` is twice in the output

Reply:
Is it a problem?